lib/protocol/SqlString.js

```diff
@@ -57,11 +59,18 @@ SqlString.escape = function(val, stringifyObjects, timeZone) {
   return "'"+val+"'";
 };
 
-SqlString.arrayToList = function(array, timeZone) {
-  return array.map(function(v) {
-    if (Array.isArray(v)) return '(' + SqlString.arrayToList(v, timeZone) + ')';
-    return SqlString.escape(v, true, timeZone);
-  }).join(', ');
+SqlString.arrayToList = function arrayToList(array, timeZone) {
+  var sql = '';
+  for (var i = 0; i < array.length; i++) {
+    var v = array[i];
+    if (i) sql += ', ';
+    if (Array.isArray(v)) {
+      sql += '(' + arrayToList(v, timeZone) + ')';
```

Comment:
The change from `SqlString.arrayToList` to just `arrayToList` will affect people who are using AOP around this function, since we have been calling it as a method. Just noting this if you wonder why that part is reverted in the merge of your pull request.